ykrt/try_passes.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+
+import argparse
+import os, sys
+from subprocess import Popen, PIPE
+from dataclasses import dataclass
+import math
+import shutil
+from multiprocessing import Process, Manager, Pool, Lock
+import time
+
+# Stages in an LTO pipeline where optimisation passes can happen.
+STAGES = "pre_link", "link_time"
```

Comment:
This isn't valid Python syntax is it? Or is it and I just don't know it?!

Reply:
Parens are optional in tuples.

```
>>> X = 1, 2
>>> X
(1, 2)
```

(I think I wrote this line.)

Should `link_time` be `post-link` to be consistent with LLVM? We should add a comment to explain their confusing terminology.

Reply:
I did not know that!